CAIPs/caip-341.md

```diff
@@ -0,0 +1,119 @@
+---
+caip: 341
+title: Extension ID Target Type Specification
+author: [Joao Tavares] (@ffmcgee725)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/341
+status: Draft
+type: Standard
+created: 2024-12-12
+requires: 294
+---
+
+## Simple Summary
+
+CAIP-341 defines the `extensionId` type as a valid target type for establishing connections with browser extension wallets.
+
+## Abstract
+
+This proposal introduces a new target type `extensionId` for the `target` field in the `walletData` interface. This target type is used to specify the extension ID of a browser extension wallet, allowing Decentralized Applications (dapps) to establish connections with the wallet using the `externally_connectable` API.
+
+## Motivation
+
+Currently, Decentralized Applications need to support different messaging standards for each namespace, such as Ethereum's [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963), Solana Wallet Protocol, etc. This fragmentation increases complexity and reduces interoperability. By defining a standardized target type for browser extension wallets, we aim to simplify the connection process and enhance interoperability across different blockchain ecosystems.
```

Comment:
```suggestion
CAIP-294 proposes a solution to fragmentation across blockchain ecosystems wallet discovery mechanisms (e.g  Ethereum's [EIP-6963], Solana's [Wallet Standard]). By defining a standardized target type for browser extension wallets that use the `externally_connectable` browser API, we aim to extend CAIP-294's unified solution to cross ecosystem wallet discoverability, enhancing interoperability across these different blockchain ecosystems.
```

Reply:
resolved in [5b0b6f4](https://github.com/ChainAgnostic/CAIPs/pull/341/commits/5b0b6f465b810ce42d316705db7f1e52302aa7cf)